backend/capellacollab/sessions/operators/k8s.py

```diff
@@ -120,12 +120,33 @@ def validate(self) -> bool:
             return False
 
     def create_public_route(
-        self, session_id: str, host: str, path: str, port: int
+        self,
+        session_id: str,
+        host: str,
+        path: str,
+        port: int,
+        wildcard_host: bool | None = False,
     ):
+        """Create a public route for the session
+
+        Parameters
+        ==========
+        session_id: str
+            The database ID of the session
+        host: str
+            The host to use for the route
+        path: str
+            The path to use for the route
+        port: int
+            The port to use for the route
+        wildcard_host: bool
+            Whether to use a wildcard host or not (serve on all hosts),
+            not support for OpenShift
```

Comment:
```suggestion
            not supported by OpenShift
```

Reply:
OpenShift might support it, but we don't support it if OpenShift is used as cluster type.

Reply:
Then let's change it to
```suggestion
            not supported for OpenShift
```

Reply:
Ah right, I've overlooked the typo. Thanks for the hint.